### PR TITLE
adjust ADB setup. fixes #5949

### PR DESCRIPTION
**Pull request by Jonahss**

### CODE REVIEW

File: lib/android-helpers.js [hunk 1]
```diff
@@ -76,23 +77,41 @@ helpers.ensureDeviceLocale = async function (adb, language, locale) {
...
+    udid = devices[0].udid;
+    emPort = adb.getPortFromEmulatorString(udid);
+  }
+  logger.info(`Using device: ${udid}`);
+  return {udid, emPort};
+};
+
+// returns a new adb instance with deviceId set
+helpers.createADB = async function (javaVersion, udid, emPort) {
+  let adb = await ADB.createADB({ javaVersion });
```
Comment (jlipps) [inline lib/android-helpers.js]: we don't use spaces in regular object like this

File: lib/driver.js [hunk 2]
```diff
@@ -92,8 +91,13 @@ class AndroidDriver extends BaseDriver {
         log.info(`Chrome-type package and activity are ${pkg} and ${activity}`);
       }
 
+      // get device udid for this session
+      let { udid, emPort } = await helpers.getDeviceInfoFromCaps(this.opts);
```
Comment (jlipps) [inline lib/driver.js]: same here

**Comment by jlipps** [comment]
couple style comments. otherwise LGTM and \o/ for tests

File: lib/android-helpers.js (hunk 1 above)
Comment (Jonahss) [inline lib/android-helpers.js]: oh gothca. Was wondering if we treated objects differently when we used the "key is same as value" shorthand.

Comment (Jonahss) [inline lib/android-helpers.js]: can we have this added to our linters?

**Comment by jlipps** [comment]
@Jonahss i'm not aware of a lint setting for this style rule

**Comment by jlipps** [comment]
LGTM now

**Pull request merged**